backend/api/users.py

```diff
@@ -185,20 +185,98 @@ async def delete_user(user: models.User = Depends(get_current_user)):
                             detail="User not found")
 
 
-@router.get("/username/{username}", status_code=status.HTTP_200_OK,
-            response_model=models.UserSummary)
-def get_user_by_username(username: str):
+@router.post("/get_by_usernames", status_code=status.HTTP_200_OK,
```

Comment:
The URI doesn't follow the standard naming conventions

Reply:
removed the function